src/utils/mockNextJsConfig.ts

```diff
@@ -20,7 +20,6 @@ export const mockDefaultConfig = () => {
       oidcClientSecret: 'secret',
       oidcApiTokensUrl: 'https://tunnistamo-backend:8000/api-tokens',
```

Comment:
Although this is just a mock, perhaps it would be best to replace also this URL and `oidcIssuer` with KeyCloak-specific values

Reply:
Changed